definitions/features/service.rb

```diff
@@ -99,9 +99,19 @@ def allowed_action?(action)
     %w[start stop restart status enable disable].include?(action)
   end
 
-  def filter_services(service_list, options)
-    service_list = include_unregistered_services(service_list, options[:include])
+  def extend_service_list_with_sockets(service_list, options)
+    return service_list unless options[:include_sockets]
 
+    if (options[:only].empty? && options[:exclude].empty?) ||
```

Comment:
This if statement runs in all conditions of service command ?
~~~
service [all]
service --only [some-service]
service --exclude [some-service]
~~~
In this case is it good to remove the if statement ?

Reply:
Good Catch! fixed